### PR TITLE
519 connect news feature to api 3

### LINKED ISSUE #519: Connect news feature to api
Call api instead of mock json file
Including correct language code

### LINKED ISSUE #697: Unionize news component with activity log components
Activity log entries should be like the new news items

https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?type=design&node-id=4350-26250&mode=design&t=3qY3hYPTMn0vhq6U-0

### LINKED ISSUE #726: Loading more mutual activities resets current scroll
When clicking on loading more, the view reloads all list of activities resetting the page scroll and having to scroll down again to see last mutual activities

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

**Comment by mikozet** [comment]
- [x] Remove filters on CONNECTIONS and NEWS
- [x] Apply our template styles in news container for components which will be added there
- [x] id of icon as a number (1, 2 or 3) - corresponding icon images 1, 2, 3 below in image
Screenshot from 2023-09-07 17-12-51
- [x] update shadow of Activity item
- [x] update paddings
- [x] correct loading more button behaviour
- [x] deployed to https://staging.circles.garden/
- [x] example of news message (can be any valid html):
```<p>Duis fugiat aliquip deserunt esse eu. Ut culpa sit adipisicing dolor tempor ex ea. Incididunt dolor labore ullamco ad magna. Irure in amet adipisicing non sit ut. Dolore in ipsum sint eiusmod nostrud. Quis excepteur ipsum tempor nisi sit mollit enim nulla nisi.</p>
<p>Lorem id quis culpa tempor do. Culpa aliquip voluptate nulla ea fugiat cupidatat laborum qui reprehenderit dolore irure dolor dolor. Culpa cillum commodo commodo dolor. Consectetur fugiat in eu quis ullamco mollit et nulla enim dolore aliquip ut duis incididunt. Occaecat mollit dolor enim commodo occaecat nostrud dolor mollit officia fugiat consectetur sint laboris elit.
  <a rel="noopener noreferrer" target="_blank" href="https://joincircles.net/faq">Learn more.</a>
</p>```

**Review by juanenrisley — APPROVED** [review]
👍🏿 LGTM

File: src/components/Footer.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import React from 'react';
 
 const Footer = (props) => {
   return (
-    <Box component="footer" mt="auto" p={2}>
+    <Box component="footer" mt="auto" p={'32px'}>
```
Comment (louilinn) [inline src/components/Footer.js:7]: What is this related to?

File: src/styles/theme.js [hunk 2]
```diff
@@ -125,6 +125,28 @@ const components = {
   navigationWidth: 300,
 };
 
+const body4Styles = {
```
Comment (louilinn) [inline src/styles/theme.js:128]: Something might have gone wrong in themes.js merging in general, fonts are off in many bodys and in balance display (see preview)

**Review by louilinn — COMMENTED** [review]
I have looked at the code semi-thouroghly and then tested in preview. Code seems good in general but found some problems

- Typography styling is broken in several places, look into this
- There are no news items displayed atm

File: src/components/Footer.js (hunk 1 above)
Comment (mikozet) [reply src/components/Footer.js:7]: That's add a padding add bottom page so items have some spacing and are not glued to the bottom

File: src/styles/theme.js (hunk 2 above)
Comment (mikozet) [reply src/styles/theme.js:128]: Can you give some examples? I built it again and it looks ok..

**Comment by mikozet** [comment]
> I have looked at the code semi-thouroghly and then tested in preview. Code seems good in general but found some problems
> 
> * Typography styling is broken in several places, look into this
> * There are no news items displayed atm

News items are available in staging environment...

**Review by louilinn — APPROVED** [review]
